docfx_yaml/extension.py

```diff
@@ -1260,6 +1261,117 @@ def find_markdown_pages(app, outdir):
             })
 
 
+# Finds and replaces occurrences which should be a cross reference in the given
+# content, except for the current name.
+def convert_cross_references(content, current_name, sorted_map):
+    words = content.split(" ")
+    new_words = []
+    # Using counter to check if the entry is already a cross reference.
+    for index, word in enumerate(words):
+        cross_reference = ""
+        for keyword in sorted_map:
+            if keyword != current_name and keyword not in current_name and keyword in word:
+                # If it is already processed as cross reference, skip over it.
+                if "<xref" in words[index-1] or (new_words and f"<xref uid=\"{keyword}" in new_words[-1]):
```

Comment:
When would this first `if` condition be true and the second be false?

* `"<xref" in words[index-1]`
* `... and f"<xref uid=\"{keyword}" in new_words[-1]`

Reply:
Ahh. Re-reading... `words` vs `new_words`. So, yes, they are different. Ignore me. :smile:

Reply:
👍